django/db/backends/mysql/features.py

```diff
@@ -319,6 +319,8 @@ def supports_index_column_ordering(self):
         return (
             not self.connection.mysql_is_mariadb
             and self.connection.mysql_version >= (8, 0, 1)
+            or self.connection.mysql_is_mariadb
+            and self.connection.mysql_version >= (10, 8, 0)
```

Comment:
Trailing zeros are unnecessary. Also, it'd be more readable to keep MySQL and MariaDB in separate branches, e.g.:
```python
if self.connection.mysql_is_mariadb:
    return self.connection.mysql_version >= (10, 8)
return self.connection.mysql_version >= (8, 0, 1)
```

Reply:
That's solid advice, I will implement it. Thanks for reviewing. :smiley: